bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ZA2JToZA2L2J_EWK_Mjj120_5f_LO/ZA2JToZA2L2J_EWK_Mjj120_5f_LO_run_card.dat

```diff
@@ -0,0 +1,270 @@
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+  0    =  nhel           ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+  20.0    =  ptj        ! minimum pt for the jets 
+  20.0    =  ptb        ! minimum pt for the b 
+  10.0    =  pta        ! minimum pt for the photons 
+  0.0    =  ptl        ! minimum pt for the charged leptons 
+  0.0    =  misset     ! minimum missing Et (sum of neutrino's momenta)
+  0.0    =  ptheavy    ! minimum pt for one heavy final state
+  1.0   = ptonium     ! minimum pt for the quarkonium states
+  -1.0    =  ptjmax     ! maximum pt for the jets
+  -1.0    =  ptbmax     ! maximum pt for the b
+  -1.0    =  ptamax     ! maximum pt for the photons
+  -1.0    =  ptlmax     ! maximum pt for the charged leptons
+  -1.0    =  missetmax  ! maximum missing Et (sum of neutrino's momenta)
+#*********************************************************************
+# Minimum and maximum E's (in the center of mass frame)              *
+#*********************************************************************
+  0.0    =  ej      ! minimum E for the jets 
+  0.0    =  eb      ! minimum E for the b 
+  0.0    =  ea      ! minimum E for the photons 
+  0.0    =  el      ! minimum E for the charged leptons 
+  -1.0    =  ejmax  ! maximum E for the jets
+  -1.0    =  ebmax  ! maximum E for the b
+  -1.0    =  eamax  ! maximum E for the photons
+  -1.0    =  elmax  ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+  6.5    =  etaj     ! max rap for the jets 
+  6.5    =  etab     ! max rap for the b
+  3.0    =  etaa     ! max rap for the photons 
+  -1.0    =  etal     ! max rap for the charged leptons 
+  0.6   = etaonium  ! max rap for the quarkonium states
+  0.0    =  etajmin  ! min rap for the jets
+  0.0    =  etabmin  ! min rap for the b
+  0.0    =  etaamin  ! min rap for the photons
+  0.0    =  etalmin  ! main rap for the charged leptons
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+  0.3    =  drjj     ! min distance between jets 
+  0.0    =  drbb     ! min distance between b's 
+  0.4    =  drll     ! min distance between leptons 
```

Comment:
what's the drll cut you will use in reco level

Reply:
We plan to apply a reco cut around the Z boson mass. For now above 80 GeV, but probably tighter in the future.